packages/astro/components/ViewTransitions.astro

```diff
@@ -3,13 +3,18 @@ type Fallback = 'none' | 'animate' | 'swap';
 
 export interface Props {
 	fallback?: Fallback;
+	form?: boolean;
```

Comment:
I like the idea of using these props to select deviant behavior.
Looking at the uses, `handleForms` seems more intuitive?

Reply:
Or maybe `animateForms` or something similar?  Astro doesn't handle anything other than catching the submit and passing everything to the endpoint. "Handle" could be misunderstood like validation, error management, etc.

Reply:
We don't use the word `animate` as animations are a use-case but not what the feature actually does. You can actually use this router and have no animations at all.